app/src/main/java/fr/free/nrw/commons/media/MediaDetailInterface.java

```diff
@@ -0,0 +1,34 @@
+package fr.free.nrw.commons.media;
+
+import com.google.gson.JsonObject;
+
+import org.wikipedia.dataclient.mwapi.MwQueryResponse;
+
+import io.reactivex.Observable;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+/**
+ * Interface for interacting with Commons Structured Data related APIs
+ */
+public interface MediaDetailInterface {
+
+    /**
+     * Fetches caption using file name
+     *
+     * @param filename name of the file to be used for fetching captions
+     * */
+
+    //Please note that languages=en does not have an impact on the languages returned. All captions are returned for all languages.
```

Comment:
please include that in the `/** */` javadoc

Reply:
done